sos/report/plugins/ceph_common.py

```diff
@@ -57,8 +57,8 @@ def setup(self):
             self.add_copy_spec("/var/log/calamari",)
 
         self.add_copy_spec([
-            "/var/log/ceph/ceph.log",
-            "/var/log/ceph/ceph.audit.log*",
+            "/var/log/ceph/**/ceph.log",
+            "/var/log/ceph/**/ceph.audit.log*",
```

Comment:
Yes I initially thought those changes were required. But I couldn't find those paths in cephadm deployments actually.

It's a safe enough change so no objection from me.